gammapy/image/models/models.py

```diff
@@ -331,9 +335,83 @@ def evaluate(x, y, amplitude, x_0, y_0):
         return x_val * y_val * amplitude
 
 
+class Template2D(Fittable2DModel):
+    """
+    Two dimensional table model .
+
+    Parameters
+    ----------
+    amplitude : float
+        Amplitude of the template model.
+
+    See Also
+    --------
+    Shell2D, Sphere2D, `~astropy.modeling.models.Gaussian2D`
+
+    """
+    amplitude = Parameter('amplitude')
+
+    def __init__(self, image, amplitude=1., **constraints):
+        self.image = image
+        super(Template2D, self).__init__(amplitude=amplitude, **constraints)
+
+    @lazyproperty
+    def _interpolate_data(self):
+        """Interpolate data using `~scipy.interpolate.RegularGridInterpolator`."""
+        from scipy.interpolate import RegularGridInterpolator
+        #TODO: move e.g. to SkyImage.interpolate()
+
+        y = np.arange(self.image.data.shape[0])
+        x = np.arange(self.image.data.shape[1])
+
+        data_normed = self.image.data / self.image.data.sum()
+        data_normed /= self.image.solid_angle().to('deg2').value
+
+        f = RegularGridInterpolator((y, x), data_normed, fill_value=0,
+                                    bounds_error=False)
+
+        def interpolate(y, x, method='linear'):
+            shape = y.shape
+            coords = np.column_stack([y.flat, x.flat])
+            val = f(coords, method=method)
+            return val.reshape(shape)
+
+        return interpolate
+
+    @classmethod
+    def read(cls, filename):
+        """
+        Read spatial template model from fits image.
+
+        Parameters
+        ----------
+        filename : str
+            Fits image filename.
+        """
+        template = SkyImage.read(filename)
+        return cls(template)
+
+    def evaluate(self, x, y, amplitude):
+        coord = SkyCoord(x, y, frame='galactic', unit='deg')
+        x_pix, y_pix = self.image.wcs_skycoord_to_pixel(coord)
+        values = self._interpolate_data(y_pix, x_pix)
+        return amplitude * values
+
+    @property
+    def bounding_box(self):
```

Comment:
Move `bounding_box` to the `SkyImage` class?
Should be useful in other contents as well, no?

This is what we have:
http://docs.gammapy.org/en/latest/api/gammapy.image.SkyImage.html#gammapy.image.SkyImage.footprint
I thought this just computes the corners and that's what the docstring says.
Here I see that also e.g. center is computed.
@adonath - Did you add this?
I think this is bad, there can just be separate properties like center, or whatever is needed, and footprint should just return what the name says: the footprint (not center, width, height).